src/main/java/com/google/codeu/servlets/ForumServlet.java

```diff
@@ -91,18 +66,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             return;
         }
 
-        String[] continents = {"Africa", "America", "Asia", "Europe", "Australia"};
+        String title = Jsoup.clean(request.getParameter("title"), Whitelist.none());
```

Comment:
I think users can make forums with same name, so it can break the datastore.getForumByName() method.
I don't mind forums having the same name, and it won't overwrite other forums with same name because forums have UUID. We probably have to protect the default forums.